contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2690,7 +2694,11 @@ rewriteBatchLevelStatement(
 			{
 				auto options = cctx->function_option();
 				auto commas = cctx->COMMA();
-				GetTokenFunc<TSqlParser::Function_optionContext*> getToken = [](TSqlParser::Function_optionContext* o) { return o->SCHEMABINDING(); };
+				GetTokenFunc<TSqlParser::Function_optionContext*> getToken = [](TSqlParser::Function_optionContext* o) {
```

Comment:
The above comment `CREATE FUNCTION ... RETURNS TABLE RETURN SELECT` looks incorrect. Please fix that as well for other's convenience.

Reply:
Modified in next revision by changing the comment to `CREATE FUNCTION ... RETURNS TABLE(COLUMN_DEFINITION) AS BEGIN ... END` as its more suited for functions of return table which was declared as
```
func_body_returns_table
    : RETURNS LOCAL_ID table_type_definition
        (WITH function_option (COMMA function_option)*)?
        AS?
        BEGIN sql_clauses* SEMI? END
    ;
```